### PR TITLE
Adding integration test's product specific scripts

### PR DESCRIPTION
**Pull request by kavithasub**
## Purpose
To have product specific integration test scripts which would get all common methods and constant variables imports from common scripts

## Approach
Prepare product specific scripts as below
- Product specific constants variables - 'prod_test_constant.py'
- Product specific implementation and main method - 'prod_test_manager.py' - this will be invokable from intg-test-runner.sh

### CODE REVIEW

File: integration-tests/prod_test_constant.py [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+                                            'wso2/business-process/dbscripts/bps/bpmn/create/activiti.mssql.create.engine.sql',
+                                            'wso2/business-process/dbscripts/bps/bpmn/create/activiti.mssql.create.history.sql',
+                                            'wso2/business-process/dbscripts/bps/bpmn/create/activiti.mssql.create.identity.sql',
+                                            'wso2/business-process/dbscripts/bps/bpmn/create/activiti.mssql.create.substitute.sql']}}},
+
+    "ORACLE-SE2": {"prefix": "jdbc:oracle:thin:@", "driverClassName": "oracle.jdbc.OracleDriver",
+                   "jarName": "oracle-se.jar",
+                   "DB_SETUP": {
+                       "product-apim": {},
+                       "product-is": {},
```
Comment (kasunbg) [inline integration-tests/prod_test_constant.py]: Why are these there? shall we remove please.

File: integration-tests/prod_test_manager.py [hunk 2]
```diff
@@ -0,0 +1,303 @@
...
+        save_test_output()
+        cm.create_output_property_fle()
+    except Exception as e:
+        logger.error("Error occurred while running the run-intg.py script", exc_info=True)
+    except BaseException as e:
+        logger.error("Error occurred while doing the configuration", exc_info=True)
+
+
+if __name__ == "__main__":
+    main()
```
Comment (kasunbg) [inline integration-tests/prod_test_manager.py]: need new line at eof.

File: integration-tests/prod_test_constant.py (hunk 1 above)
Comment (kavithasub) [reply integration-tests/prod_test_constant.py]: Removed product ids and fixed in 5b5b94186229d940cfe147d6a2a8071b0d0320de

File: integration-tests/prod_test_manager.py (hunk 2 above)
Comment (kavithasub) [reply integration-tests/prod_test_manager.py]: Fix for new line in 5b5b94186229d940cfe147d6a2a8071b0d0320de

**Comment by kavithasub** [comment]
It is complex to resolve conflict, hence closing this PR and sending a new one.

**Pull request closed without merging**